doc/release_notes/release_notes_2.5.rst

```diff
@@ -57,93 +57,67 @@ Upgrading to v2.5 From Previous Releases
 We highly recommended that you follow the instructions below to
 upgrade to v2.5 from previous ACRN releases.
 
-Additional Dependencies
-=======================
+Generate New Board XML
+======================
 
-Python version 3.6 or higher is required to build ACRN v2.5. You can check the version of
-Python you are using by:
+Board XML files, generated by ACRN board inspector, contain board information
+that is essential to build ACRN. Compared to previous versions, ACRN v2.5
+extends the schema of board XMLs to summarize board information more
+systematically. It is thus required to regenerate the board XML using the new
+board inspector when you upgrade to ACRN v2.5.
 
-.. code-block:: bash
-
-   $ python3 --version
-   Python 3.5.2
-
-Only when the reported version is less than 3.6 (as is the case in the example above) do
-you need an upgrade. The first (and preferred) choice is to install the latest
-Python 3 from the official package repository:
+To use the new board inspector, ensure you have Python >= 3.6 on the target
+board with the package lxml installed. Refer to :ref:`upgrade_python` for
+detailed steps to check and upgrade Python version. The lxml package can be
+installed by executing the following command:
 
 .. code-block:: bash
 
-   $ sudo apt install python3
-   ...
-   $ python --version
-   Python 3.8.8
+   $ sudo pip3 install lxml
```

Comment:
```suggestion
   sudo pip3 install lxml
```

Reply:
Typically I use ``$`` to indicate that the line is a command to be executed. May I know the overall guideline on writing such snippets?

Reply:
> Typically I use `$` to indicate that the line is a command to be executed. May I know the overall guideline on writing such snippets?

This is also what we've done for a long time. The problem with this approach is that people cannot copy-paste the instructions as it also copies the `$` symbol. I do not believe we have documented this anywhere but we tend to only write the command without the leading prompt (it being `$` or `#`).

I believe that in the Zephyr Project, the HTML content that gets generated gets such prompt added to those code blocks automatically, and in a way that they do not get copied when people copy-paste them.

@dbkinder can correct me or provide more details on that I guess ;-)